### PR TITLE
RDoc-642: add linux recommendation page

### PR DESCRIPTION
**Pull request by aviviadi**

### CODE REVIEW

File: Documentation/4.0/Raven.Documentation.Pages/start/linux-config-recommandations.markdown [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+* Consider the following setup. 
+* Note that it might affect different system with a different behaivour.
+* Some of the settings requires super user permissions
+```
+ulimit -n 65535
+sysctl -w net.ipv4.ip_local_port_range="1024 65535"
+sysctl -w vm.max_map_count=2000000
+sysctl -w net.ipv4.tcp_tw_reuse=1
+sysctl -w net.ipv4.tcp_tw_recycle=1
+```
```
Comment (ayende) [inline Documentation/4.0/Raven.Documentation.Pages/start/linux-config-recommandations.markdown]: Add some explanation on why we need these

File: Documentation/4.0/Raven.Documentation.Pages/start/linux-config-recommandations.markdown [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Linux Configuration Recommendations With RavenDB
+RavenDB uses the resources of the machine it is running at, limited to the configuration limitation. In order to benfite higher resources usage, consider the following setup:
+
+* Note that it might affect different system with a different behaivour.
```
Comment (ppekrol) [inline Documentation/4.0/Raven.Documentation.Pages/start/linux-config-recommandations.markdown]: Can we put this in a {NOTE} text {NOTE/} block?

File: Documentation/4.0/Raven.Documentation.Pages/start/linux-config-recommandations.markdown [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+`sysctl -w net.ipv4.tcp_tw_recycle=1`
+##### Enlarge memory mapped max usage
+RavenDB uses memory mapped files (few per database, indexes, etc). Upper limit is required in order to server bigger amount of databases and indexes
+
+`sysctl -w vm.max_map_count=2000000`
+##### Select swap device
+Consider different swap physical drive.
+For details on current swapping partitions and priorities use:
+
+`swapon`
```
Comment (ppekrol) [inline Documentation/4.0/Raven.Documentation.Pages/start/linux-config-recommandations.markdown]: I think that we are missing a description here?

File: Documentation/4.0/Raven.Documentation.Pages/start/linux-config-recommandations.markdown (hunk 2 above)
Comment (aviviadi) [reply Documentation/4.0/Raven.Documentation.Pages/start/linux-config-recommandations.markdown]: My MD editor doesn't applies {NOTE} in any manner.

**Pull request merged**